.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,6 @@
 * [ ] Create a file in `src/towncrier/newsfragments/`. Describe your
   change and include important information. Your change will be included in the public release notes.
 * [ ] Make sure all GitHub Actions checks are green (they are automatically checking all of the above).
+* [ ] Ensure `docs/tutorial.rst` is still up-to-date.
+* [ ] If you add new **CLI arguments** (or change the meaning of existing ones), make sure `docs/cli.rst` reflects those changes.
```

Comment:
I think this ok for now.

In the future maybe we can use something like https://pypi.org/project/sphinx-click/ to automatically keep the reference up to date.

Reply:
Yes and no…I think the reference docs should do more than parroting `--help` output. Maybe some kind of combination idk.